qa/workunits/rbd/rbd_mirror_helpers.sh

```diff
@@ -264,21 +264,21 @@ setup_pools()
         rbd --cluster ${cluster} mirror pool peer add ${PARENT_POOL} ${remote_cluster}
       else
         mon_map_file=${TEMPDIR}/${remote_cluster}.monmap
-        ceph --cluster ${remote_cluster} mon getmap > ${mon_map_file}
+        CEPH_ARGS='' ceph --cluster ${remote_cluster} mon getmap > ${mon_map_file}
         mon_addr=$(monmaptool --print ${mon_map_file} | grep -E 'mon\.' |
           head -n 1 | sed -E 's/^[0-9]+: ([^ ]+).+$/\1/' | sed -E 's/\/[0-9]+//g')
 
         admin_key_file=${TEMPDIR}/${remote_cluster}.client.${CEPH_ID}.key
         CEPH_ARGS='' ceph --cluster ${remote_cluster} auth get-key client.${CEPH_ID} > ${admin_key_file}
 
-        rbd --cluster ${cluster} mirror pool peer add ${POOL} \
+        CEPH_ARGS='' rbd --cluster ${cluster} mirror pool peer add ${POOL} \
             client.${CEPH_ID}@${remote_cluster}${PEER_CLUSTER_SUFFIX} \
             --remote-mon-host "${mon_addr}" --remote-key-file ${admin_key_file}
 
         uuid=$(rbd --cluster ${cluster} mirror pool peer add ${PARENT_POOL} \
             client.${CEPH_ID}@${remote_cluster}${PEER_CLUSTER_SUFFIX})
```

Comment:
@dillaman Does the "rbd mirror pool peer add" command in the `uuid` assignment line need `CEPH_ARGS=''``, too?

Reply:
Negative -- it doesn't attempt to store data in the MON